extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -127,263 +141,509 @@ For example
-3. A property texture is associated with a primitive by listing the property texture ID in the `primitive.EXT_mesh_features.propertyTextures` array.
-<img src="figures/feature-texture.png"  alt="Property Texture" width="500">
-
-_Class and property texture_
-
-```jsonc
-{
-  "extensions": {
-    "EXT_mesh_features": {
-      "schema": {
-        "classes": {
-          "heatSample": {
-            "properties": {
-              "heatLoss": {
-                "type": "UINT8",
-                "normalized": true
-              },
-              "insulation": {
-                "type": "UINT8",
-                "normalized": true
-              },
-            }
-          }
-        }
-      },
-      "propertyTextures": [{
-        "class": "heatSample",
-        "index": 0,
-        "texCoord": 0,
-        "properties": {
-          "heatLoss": [0],
-          "insulation": [1]
-        }
-      }]
-    }
-  }
-}
-```
-
-_Primitive_
-
-```jsonc
-{
-  "primitives": [
-    {
-      "attributes": {
-        "POSITION": 0,
-        "TEXCOORD_0": 1
-      },
-      "indices": 2,
-      "material": 0,
-      "extensions": {
-        "EXT_mesh_features": {
-          "propertyTextures": [0]
-        }
-      }
-    }
-  ]
-}
-```
-
-
-A `propertyTexture` object extends the glTF [`textureInfo`](../../../../../specification/2.0/schema/textureInfo.schema.json) object. `texCoord` refers to the texture coordinate set of the referring primitive. The `properties` map specifies the texture channels providing data for all required class properties, and perhaps optional class properties. An array of integer index values identify channels, where multiple channels may be used only for fixed-length arrays of 2, 3, or 4 components. Channels of an `RGBA` texture are numbered 0–3 respectively, although specialized texture formats may allow additional channels. All values are stored in linear space.
+* Data type and bit depth of the image must be compatible with the property type.
+
+For example, an 8-bit per pixel RGB image may only contain `SINGLE`, fixed-length `ARRAY` (length ≤3), `VEC2`, or `VEC3` values composed of 8-bit component types.
+
+Enum values may be encoded in images, as integer values according to their enum value type (see [Enum](#enum)).
+
+> **Implementation note:** Use of floating-point properties in a property texture would require a floating point-compatible image format like KTX2, provided by an additional extension.
+
+> **Example:** Property texture implementing a "wall" class, with property values stored in a glTF texture at index 0 and indexed by `TEXCOORD_0`.
+>
+> <img src="figures/feature-texture.png"  alt="Property Texture" width="500">
+>
+> _Class and property texture_
+>
+> ```jsonc
+> {
+>   "extensions": {
+>     "EXT_mesh_features": {
+>       "schema": {
+>         "classes": {
+>           "wall": {
+>             "name": "Wall Temperature vs. Insulation",
+>             "properties": {
+>               "insideTemp": {
```

Comment:
Oh I think I glossed over this in a previous review pass, this should be `insideTemperature`. We like to be consistent with our [CesiumJS style guide](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/CodingGuide#naming) where possible for consistency.

Reply:
Done ✅